site-src/api-types/gatewayclass.md

```diff
@@ -136,4 +136,4 @@ acme.io/gateway      // Use the default version
 ```
 
 [gatewayclass]: https://gateway-api.sigs.k8s.io/references/spec/#networking.x-k8s.io/v1alpha1.GatewayClass
-[ingress-class-api]: https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/20190125-ingress-api-group.md#ingressclass-resource
+[ingress-class-api]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/758-ingress-api-group#ingressclass-resource
```

Comment:
Thanks for fixing this! While we're at it, I think it would be better to link to the docs than the KEP: https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-class.

Reply:
Done